megamek/src/megamek/client/bot/princess/FireControl.java

```diff
@@ -2598,7 +2598,8 @@ FiringPlan getBestFiringPlan(final Entity shooter,
                                                                              ammoConservation);
             final FiringPlan plan = determineBestFiringPlan(parameters);
 
-            LogManager.getLogger().info(shooter.getDisplayName() + " at " + enemy
+            // Debug logging; Princess does her own info logging
+            LogManager.getLogger().debug(shooter.getDisplayName() + " at " + enemy
```

Comment:
General optimization suggestion: for methods that are called many times, calling logging methods (especially within a loop) leads to a performance decrease, even if the logging level doesn't result in the message being logged.  Let's make the adjustment to comment this out as we don't really need the info outside of direct debugging.

Reply:
I think we can just do the debug logging alongside the Princess-level .info() logging, which is once per unit per round.

Reply:
Sure. This logging is is once per unit per enemy per round. Granted, it's during the firing phase which doesn't really have performance issues usually, but still.